### PR TITLE
refactor(query): enable new planner rules and remove old rules

### PR DESCRIPTION
**Pull request by nathanielc**

### CODE REVIEW

File: query/stdlib/influxdata/influxdb/from.go [hunk 1]
```diff
@@ -38,14 +37,6 @@ func init() {
 	flux.ReplacePackageValue("influxdata/influxdb", influxdb.FromKind, flux.FunctionValue(FromKind, createFromOpSpec, fromSignature))
 	flux.RegisterOpSpec(FromKind, newFromOp)
 	plan.RegisterProcedureSpec(FromKind, newFromProcedure, FromKind)
-	plan.RegisterPhysicalRules(
-		FromConversionRule{},
-		MergeFromRangeRule{},
-		MergeFromFilterRule{},
-		FromDistinctRule{},
-		MergeFromGroupRule{},
```
Comment (jsternberg) [inline query/stdlib/influxdata/influxdb/from.go]: Are we just not going to push down group rules? I'm fine with that, but I just want to confirm that's intended.

Comment (nathanielc) [reply query/stdlib/influxdata/influxdb/from.go]: I thought it was incompatible with the new rules? Is that not the case?

Comment (jsternberg) [reply query/stdlib/influxdata/influxdb/from.go]: It is. I just wanted to confirm that, at least temporarily, this was ok. We should create an issue to redo this rule with the new rules also.

Comment (jlapacik) [reply query/stdlib/influxdata/influxdb/from.go]: @nathanielc that is correct, none of the old rules are compatible.
@jsternberg Group will need to be a separate PR.

File: query/stdlib/influxdata/influxdb/rules.go [hunk 2]
```diff
@@ -2,20 +2,21 @@ package influxdb
...
-// 		PushDownFilterRule{},
-// 		PushDownReadTagKeysRule{},
-// 		PushDownReadTagValuesRule{},
-// 	)
-// }
+func init() {
+	plan.RegisterPhysicalRules(
+		PushDownRangeRule{},
+		PushDownFilterRule{},
+		//PushDownReadTagKeysRule{},
```
Comment (jlapacik) [inline query/stdlib/influxdata/influxdb/rules.go]: I thought we can un-comment this one too?

Comment (nathanielc) [reply query/stdlib/influxdata/influxdb/rules.go]: Its merged to master but not deployed yet. So I wanted to hold off on that one so we can deploy this change without blocking on a storage deploy.

File: query/stdlib/influxdata/influxdb/from.go (hunk 1 above)
Comment (nathanielc) [reply query/stdlib/influxdata/influxdb/from.go]: Sounds good, Issue created https://github.com/influxdata/influxdb/issues/13623

Can either of you add more details to that issue if they are needed? Also does that rule need a new RPC call to be exposed from storage?

Comment (jlapacik) [reply query/stdlib/influxdata/influxdb/from.go]: Sure I'll add it. And yes that will be the final RPC call to be exposed from storage.

**Pull request merged**